jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildContainerConfigurationStep.java

```diff
@@ -59,11 +60,11 @@
   @Override
   public ListenableFuture<Blob> call() throws ExecutionException, InterruptedException {
     // TODO: This might need to belong in BuildImageSteps.
-    List<ListenableFuture<?>> afterBaseImageLayerFuturesFutureDependencies = new ArrayList<>();
-    afterBaseImageLayerFuturesFutureDependencies.addAll(
+    List<ListenableFuture<?>> afterBaseImageLayerFutureFutureDependencies = new ArrayList<>();
```

Comment:
Oh this one is actually a future of futures (it refers to being after `pullBaseImageLayerFuturesFuture`)